src/components/atoms/Button/style.scss

```diff
@@ -31,17 +31,20 @@
     display: inline-block;
     padding: 4px;
 
-    &-left {
+    &--left {
+        @extend .arrow;
         transform: rotate(135deg);
         margin-left: 5px;
     }
 
-    &-right {
+    &--right {
+        @extend .arrow;
         transform: rotate(-45deg);
         margin-right: 5px;
     }
 
-    &-big {
+    &--big {
+        @extend .arrow;
```

Comment:
why you need to extend arrow? just use `arrow arrow--big`

Reply:
Changed